.github/dependabot.yml

```
@@ -0,0 +1,15 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  allow:
+  - dependency-type: direct
+  - dependency-type: indirect
```

Could be simplified to

```suggestion
  - dependency-type: all
```

?